test/runtests.jl

```diff
@@ -18,8 +18,15 @@ end
 
 non_numeric_arg_functions = [(:Base, :rem2pi, 2), (:Base, :ifelse, 3)]
 
+recently_defined_functions = []
+if !hasmethod(SpecialFunctions.gamma, Tuple{Number, Number})
+    # 2-arg gamma added after SpecialFunctions v0.8.0
+    push!(recently_defined_functions, (:SpecialFunctions, :gamma, 2))
+end
+
```

Comment:
We should just update the SpecialFunctions compat entry if needed. Might not be necessary anymore now that we only support SpecialFunctions >= 0.10 on the master branch?
```suggestion
```